examples/cluster/plot_digits_linkage.py

```diff
@@ -2,35 +2,28 @@
 =============================================================================
 Various Agglomerative Clustering on a 2D embedding of digits
 =============================================================================
-
 An illustration of various linkage option for agglomerative clustering on
 a 2D embedding of the digits dataset.
-
 The goal of this example is to show intuitively how the metrics behave, and
 not to find good clusters for the digits. This is why the example works on a
 2D embedding.
-
 What this example shows us is the behavior "rich getting richer" of
 agglomerative clustering that tends to create uneven cluster sizes.
-
 This behavior is pronounced for the average linkage strategy,
 that ends up with a couple of clusters with few datapoints.
-
 The case of single linkage is even more pathologic with a very
 large cluster covering most digits, an intermediate size (clean)
 cluster with most zero digits and all other clusters being drawn
 from noise points around the fringes.
-
 The other linkage strategies lead to more evenly distributed
 clusters that are therefore likely to be less sensible to a
 random resampling of the dataset.
-
 """
 
 # Authors: Gael Varoquaux
 # License: BSD 3 clause (C) INRIA 2014
 
-from time import time
```

Comment:
Let's not introduce `perf_counter` that is never used in the documentation and stick to `time`.